docker-compose.yml

```diff
@@ -9,6 +9,8 @@ services:
       context: ./nginx
     ports:
       - "8080:80"
+    links:
```

Comment:
I think it's not necessary

Reply:
Stupid GitHub diff didn't break the changes correctly. Line 12 comes together with line 13. Together they are needed so "api" service will find "mongo" service. Otherwise this was broken for me.